Readme.md

```diff
@@ -30,6 +30,19 @@ You are more than welcome to contribute to Phel. You can do so by either:
 * contributing changes
 * enrich the documentation
 
+### Substantial changes
+
+Substantial changes are architecture decisions, documentation restructuring, breaking changes, etc.
+But not Bug Reports, Bug Fixes, Unit Tests, etc.
+
+#### How to contribute a substantial change
+
+In order to make a substantial change it is a good practice to discuss the idea before implementing it.
+
+- An ADR or RFC can be proposed with an issue.
```

Comment:
For people unfamiliar with abbreviations of ADR and RFC it would be good to define them as full form first:
"Architecture Decision Record (ADR)".